backend/api.py

```diff
@@ -322,7 +322,6 @@ def save_niceties():
             # So then db.session.commit() sends the update (or insert) for every object
             # in the session. This includes every object created by a [model].query and
             # everything added to the session with db.session.add().
-        nicety.anonymous = n.get("anonymous", current_user().anonymous_by_default)
```

Comment:
I think the `nicety.anonymous` property does still need to be set, but we should default to `False` rather than the current user's `anonymous_by_default` which we are deleting.

Reply:
Good point, added the change!